config/azure/IntelMPI-v2019.x-silent.cfg

```diff
@@ -0,0 +1,26 @@
+# Accept EULA, valid values are: {accept, decline}
+ACCEPT_EULA=accept
+
+# Install mode
+INSTALL_MODE=RPM
+
+# Optional error behavior, valid values are: {yes, no}
+CONTINUE_WITH_OPTIONAL_ERROR=yes
+
+# Install location, valid values are: {/opt/intel, filepat}
+PSET_INSTALL_DIR=/opt/intel
+
+# Continue with overwrite of existing installation directory, valid values are: {yes, no}
+CONTINUE_WITH_INSTALLDIR_OVERWRITE=yes
+
+# List of components to install, valid values are: {ALL, anythingpat}
+COMPONENTS=ALL
+
+# Installation mode, valid values are: {install, modify, repair, uninstall}
+PSET_MODE=install
+
+# Intel(R) Software Improvement Program opt-in, valid values are: {yes, no}
+#INTEL_SW_IMPROVEMENT_PROGRAM_CONSENT=no
```

Comment:
I am checking with IMPI guys on the corresponding param in v2019. [Documentation]( https://software.intel.com/en-us/articles/configuration-file-format) says INTEL_SW_IMPROVEMENT_PROGRAM_CONSENT, but it throws an error during installation.